docs/README.md

```diff
@@ -22,6 +22,7 @@ Playwright is a library to automate [Chromium](https://www.chromium.org/Home), [
     - [First script](./intro.md#first-script)
     - [Core concepts](./core-concepts.md)
     - [Debugging](./debug.md)
+    - [Playwright for Python](./python.md)
```

Comment:
In your intention is to direct traffic to Python - this is not prominent enough. And if it isn't, why do we need it? :)

Reply:
Yep that would be the outcome. My intentions was more that we currently have never referenced it anywhere except in the change log on GitHub.

(Had a chat with Arjun and he was fine with it)